polars/polars-arrow/src/kernels/rolling/no_nulls/min_max.rs

```diff
@@ -18,252 +18,164 @@ impl<'a, T: NativeType> RollingAggWindowNoNulls<'a, T> for SortedMinMax<'a, T> {
     }
 }
 
+#[inline]
+unsafe fn get_min_and_ix<T>(slice: &[T], start: usize, end: usize) -> Option<(usize, &T)>
```

Comment:
nit: can you rename all `ix` to `idx` to make it more consistent with how we name indexes in the code base?